frontends/web/src/routes/exchange/components/buysell.tsx

```diff
@@ -106,40 +106,6 @@ export const BuySell = ({
       <div className={style.innerRadioButtonsContainer}>
         {!exchangeDealsResponse && <Skeleton />}
 
-        {action === 'sell' ? ( // TODO: this can be removed once backend side is updated
-          <div key={'btcdirect-sell'} className={style.actionableItemContainer}>
-            ----START Testing: BTC Direct sell
-            <ActionableItem
-              key={'btcdirect-sell'}
-              onClick={() => {
-                goToExchange('btcdirect');
-              }}>
-              <ExchangeProviders
-                deals={[{
-                  fee: 4.9,
-                  payment: 'card',
```

Comment:
sorry this was already wrong in the hardcoded option, but when selling coins there is no creditcard option.